scripts/redraw-numpy-dep-charts.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# This script refreshes all numpy dependency graphs for packages
+# that are mentioned in NumPy case studes: ehtim, gwpy and PyCBC
+#
+
+set -ue
+
+curdir=`pwd`
+outputdir=$curdir
+
+if [[ "$#" -ge "2" || (( "$#" -eq "1" ) && ( "$1" == "-h" || "$1" == "--help" )) ]]; then
+    echo ""
+    echo "Usage:  ./redraw-numpy-dep-charts.sh outputdirname <optional, default current-dir"
```

Comment:
This script actually only works when running from the root of the repo: `./scripts/redraw-numpy-dep-charts.sh`

Reply:
ok, will take care of this too.

Reply:
I'm using MacBook to test these scripts.  So far they all seem to be running fine without making any changes, yet.  Did you run them on linux?

Reply:
The new set of scripts deprecates fetch-ehtim and NumPy-ehtim-dep-graph  which were specific to generating ehtim graph only.  The new scripts are generic for specified packages albeit run in the order listed in top level script - redraw-numpy-dep-charts.sh

Reply:
Creating readme for correct usage.

Reply:
> This script actually only works when running from the root of the repo: `./scripts/redraw-numpy-dep-charts.sh`

Addressed, thanks for highlighting this bug.